docs/source/api.rst

```diff
@@ -47,3 +49,4 @@ API Reference
     api/randvars
     api/statespace
     api/utils
+    api/config
```

Comment:
just out of curiousity, nothing : why prefer `config` over `settings` as a name? :)

Reply:
Personal taste, but I don't have a strong opinion on this one. I'm happy to change it to settings, if you prefer :)

Reply:
I am 55/45 towards settings. Not sufficiently strong to ask for a change ;)

Reply:
I am 51/49 towards config, so your opinion is stronger so far 😆 Let's ask the others for their opinions

Reply:
I am 70/30 toward config, since I feel like settings is more often used for UI applications and config(uration) appears more often in lower-level contexts (as in config file)

Reply:
I'd vote for `config` as well.